### DISCUSSION TITLE
New facebook messenger icon

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2146] [Request: Icon] [answered]
Advanced message teams need a facebook messenger icon to indicate the status of the sender pool. 
Figma design here: https://www.figma.com/file/KqKh1imka3RBXjgfWSTMLc/FBM---Sender-Pool?node-id=0%3A1

I have pulled the original brand asset for the FB messenger icon from [Meta website](https://www.facebook.com/brand/resources/messenger/logo) and turn it into a outline version. 

Streamline also has an outline version but slightly different: https://app.streamlinehq.com/icons/streamline-light?search=facebook%2Bmessenger

I have created the fb messenger outlined icon in a [figma file](https://www.figma.com/file/yIf4TgbwtMEkphqTLBMaxZ/ACP-Chanel-Icon?node-id=10%3A89113) followed by the steps from Paste website and all the settings / configurations of the standard has been applied.

Same for Whatsapp icon, I have pulled it from the Meta website and have it created in [Figma](https://www.figma.com/file/yIf4TgbwtMEkphqTLBMaxZ/ACP-Chanel-Icon?node-id=11%3A89)

Does Paste need to ingest these icons to the library?

**Comment by kate-porter** [comment, marked as answer]
Hey @frankfzhao - thanks for asking about this!

We actually can't distribute branded icons such as Messenger and WhatsApp in Paste due to copyright restrictions, so what we'd recommend here is for you to pull the logos from the Meta website, as you've done, and add the icons to your Figma project, as you've also already done.

When you do this, we'd recommend following the brand standards set by Messenger and WhatsApp when using the logos in your product, even if they end up looking different from our Paste icons (for example, it doesn't look like Messenger has an outlined version of their logo, so it'd be best to use the approved filled one rather than restyling it to match Paste).

Once you've prepared the logos on your side, your eng team can use our [SVG to React library](https://github.com/twilio-labs/svg-to-react) to embed them, if it's helpful.

Let us know if you have more questions on this!

**Reply by frankfzhao** [reply]
thanks Katie!
